packages/calcite-components/src/components/input-text/input-text.tsx

```diff
@@ -585,24 +585,16 @@ export class InputText
         aria-errormessage={IDS.validationMessage}
         ariaInvalid={this.status === "invalid"}
         ariaLabel={getLabelText(this)}
-        autocomplete={this.autocomplete as LuminaJsx.HTMLElementTags["input"]["autocomplete"]}
-        autofocus={this.el.autofocus ? true : null}
+        autocomplete={this.autocomplete}
+        autofocus={this.el.autofocus}
         class={{
           [CSS.editingEnabled]: this.editingEnabled,
           [CSS.inlineChild]: !!this.inlineEditableEl,
         }}
         defaultValue={this.defaultValue}
         disabled={this.disabled ? true : null}
```

Comment:
does this still need to have `? true : null`? is that expected or is it a bug I should fix?

```suggestion
        disabled={this.disabled}
```

Reply:
I'll test for that separately to keep this PR focused.